docs-src/concepts/what-is-the-temporal-cli.md

```diff
@@ -179,18 +191,18 @@ $ temporal workflow list --fields long -o json | jq '.[].type.name' | uniq -c
    2 "MyWorkflow"
 ```
 
-To see what else you can do to Workflows, run `temporal workflow` or visit [CLI ▶️ workflow](/cli/workflow).
+To see the full range of Workflow-related commands, run `temporal workflow` or visit [CLI ▶️ workflow](/cli/workflow).
```

Comment:
```suggestion
To see the full range of Workflow-related commands, run `temporal workflow` or see the [Temporal CLI workflow command reference](/cli/workflow).
```
Per https://developers.google.com/style/cross-references#references-to-other-documents.